lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -606,15 +698,15 @@ namespace Js
                 {
                     // Done walking inlined frames in a loop body, cache the native code address now
                     // in order to skip the loop body frame.
-                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody);
+                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody, true /*inlinedFramesOnStack*/);
                     isJavascriptFrame = false;
                 }
             }
 
             return true;
         }
 #endif
-
+        this->inlinedFramesOnStack = false;
```

Comment:
in the section of code highlighted by github we have two bools isJavascriptFrame and inlinedFramesOnStack. Typically in the codebase if you want to let people know that your new variable is a boolean  you have prefix it with an is**name** or a has***condition*** . Some other code bases prefix with b. Anyways back on point, Some one else in this very function has established a precedent of using said is** naming convention. when you name a bool inlinedFramesOnStack you break the naming convention. my initial thought when reading that name was your new variable was a pointer until i saw you assign it to a bool, hence the nit. Really its not a big deal just thought I would point it out.